.github/workflows/dependency-review.yml

```diff
@@ -31,11 +31,16 @@ on:
         required: false
         default: high
         type: string
+      runs-on:
+        description: |
+          The type of machine to run the job on. Must be provided as a stringified list (e.g. `runs-on: '["ubuntu-latest","self-hosted"]'`)
+        default: '["coveo", "arm64" , "linux", "eks"]'
```

Comment:
**question**: What's the plan with workflows/actions that are duplicated across `coveo/public-actions` and `coveo/actions`?

Should we eventually use the action directly from the `coveo/public-actions` repo or should we keep using the `coveo/actions`? If we want to stay with the later, I think we should put this default on the `coveo/action` workflow as it's tied to Coveo's infrastructure.

For reference, we have this workflow https://github.com/coveo/actions/blob/main/.github/workflows/dependency-review.yml calling this one.

Reply:
As discussed in slack; the private one will be removed since it's not used enough.